lib/rbac/authorizer.rb

```diff
@@ -57,9 +48,8 @@ def user_role_allows?(user, options = {})
       end
     end
 
-    def user_role_allows_any?(user, options = {})
-      return false if user.miq_user_role.nil?
-      user.miq_user_role.allows_any?(**options)
+    def user_role_allows_any?(user, identifiers:)
+      user.miq_user_role.allows_any?(:identifiers => identifiers)
```

Comment:
Similar comment with the **options vs picking a specific one?

Reply:
yea, similar comment from me. this shows what is actually used when reading the source code.

making this change.